catalogue/webapp/pages/workv2.js

```diff
@@ -302,12 +304,10 @@ export const WorkPage = ({
                           action: 'download-button:click',
                           label: `id: $work.id} , size:760, title:${work.title.substring(50)}`
                         }}
-                        clickHandler={() => {
-                          ReactGA.event({
-                            category: 'component',
-                            action: 'download-button:click',
-                            label: `id: $work.id} , size:760, title:${work.title.substring(50)}`
-                          });
+                        trackingEventV2={{
+                          eventCategory: 'Button',
```

Comment:
I know we've agreed this, but just wondering, what happens when we change this to a `Control` or `SecondaryLink`?

It does feel, potentially as though category could be more real-life orientated, if what we're tracking is that?

e.g. `{ 'category': 'Work', 'action': 'download large image', 'label': '1234' }`

But if we wanted to track now well buttons are doing, we could also track that?